### PR TITLE
feat(skin): make border radius configurable by skin

### PR DESCRIPTION
**Pull request by atabel**
jira: https://jira.tid.es/browse/WEB-817

### CODE REVIEW

File: src/skins/telefonica.tsx [hunk 1]
```diff
@@ -264,5 +264,16 @@ export const getTelefonicaSkin: GetKnownSkin = () => {
             promoHighInverse: palette.orchid70,
             neutralMediumInverse: palette.grey5,
         },
+        borderRadii: {
```
[the remark this answers is not in the document]
Comment (brtbrt) [reply src/skins/telefonica.tsx:267]: it's the plural 🌝

File: src/progress-bar.css.ts [hunk 2]
```diff
@@ -5,14 +5,14 @@ import {style, keyframes} from '@vanilla-extract/css';
...
-    borderRadius: 2,
+    borderRadius: vars.borderRadii.indicator,
     height: 4,
     background: vars.colors.control,
 });
 
 export const bar = sprinkles({
     height: '100%',
-    borderRadius: 2,
+    borderRadius: vars.borderRadii.indicator,
```
Comment (yceballost) [inline src/progress-bar.css.ts:15]:
```suggestion
    borderRadius: vars.borderRadii.bar,
```

File: src/progress-bar.css.ts [hunk 3]
```diff
@@ -5,14 +5,14 @@ import {style, keyframes} from '@vanilla-extract/css';
 const transition = '1s cubic-bezier(0.75, 0, 0.27, 1)';
 
 export const barBackground = sprinkles({
-    borderRadius: 2,
+    borderRadius: vars.borderRadii.indicator,
```
Comment (yceballost) [inline src/progress-bar.css.ts:8]:
```suggestion
    borderRadius: vars.borderRadii.bar,
```

File: src/stepper.css.ts [hunk 4]
```diff
@@ -178,7 +178,7 @@ export const bar = style([
     sprinkles({
         height: 4,
         width: '100%',
-        borderRadius: 4,
+        borderRadius: skinVars.borderRadii.indicator,
```
Comment (yceballost) [inline src/stepper.css.ts:181]:
```suggestion
        borderRadius: skinVars.borderRadii.bar,
```

File: src/stepper.css.ts [hunk 5]
```diff
@@ -200,7 +200,7 @@ export const barFilled = style([
     sprinkles({
         height: 4,
         width: '100%',
-        borderRadius: 20,
+        borderRadius: skinVars.borderRadii.indicator,
```
Comment (yceballost) [inline src/stepper.css.ts:203]:
```suggestion
        borderRadius: skinVars.borderRadii.bar,
```

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/skins/telefonica.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (atabel) [reply src/skins/telefonica.tsx:267]: we have `colors` (plural of color), so it makes sense to use `borderRadii` (plural of borderRadius) here

File: src/sprinkles.css.ts [hunk 6]
```diff
@@ -43,7 +43,7 @@ const commonProperties = defineProperties({
             regular: `1px solid ${vars.colors.border}`,
             none: 'none',
         },
-        borderRadius: ['50%', 2, 4, 8, 16, 20],
+        borderRadius: ['50%', 2, 4, 8, 16, 20, ...Object.values(vars.borderRadii)],
```
[the remark this answers is not in the document]
Comment (atabel) [reply src/sprinkles.css.ts:46]: no, the css vars that vanilla extracts generates for those

**Review by brtbrt — APPROVED** [review]

File: src/skins/telefonica.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (yceballost) [reply src/skins/telefonica.tsx:267]: 🤯

**Review by yceballost — APPROVED** [review]

**Review by AnaMontes11 — APPROVED** [review]
Great job! ✨

**Review by marcoskolodny — APPROVED** [review]

**Pull request merged**